picard/coverart/__init__.py

```diff
@@ -130,7 +137,7 @@ def download_next_in_queue(self):
                     p = provider(self)
                     if p.enabled():
                         log.debug("Trying cover art provider %s ..." % name)
-                        ret = p.queue_downloads()
+                        ret = p.queue_images()
```

Comment:
Not sure how i should provide the compatibility, it is expected for queue_images() to be implemented, any idea on a clean way (if we keep the rename ofc) ?